README.md

```diff
@@ -2,19 +2,134 @@
 
 ## Bring 'Code as Data' to market with enterprises
 
+### Enable engine integration with 3rd parties
+
+- [ ] gitbase-spark-connector SQL 3rd party interop (Spark Thrift Server) [P0][DP]
```

Comment:
Since the section is "Enable engine integration with 3rd parties", will this be expected to be available in engine?

Reply:
Not now or not yet. Probably in next quarter, but still requires product discussions.